test/unit/polling/polling.test.js

```diff
@@ -0,0 +1,64 @@
+jest.useFakeTimers()
+jest.spyOn(global, 'setTimeout')
+
+const config = require('../../../app/config/verify')
+jest.mock('../../../app/config/verify')
```

Comment:
oorrrr is it to get the object keys tht aren't functions?

i.e. lines 14 & 15 we overwrite what `pollingInterval` and `pollingActive` are but `jest.mock` does not pick these up and only cares about functions?

Reply:
Yeah it's to get the object keys that aren't functions.  I could also have set it up so it returns a mock function that just has the keys as the mocked return value.  Then just change the mock return value for tests I want different behaviour.  Mocking modules that return objects in Jest feels like it could be better.  Everything feels a bit like a work around.